### PR TITLE
Fixes #25187 Include value for ATTACHED & QUANTITY

### PR DESCRIPTION
**Pull request by vision2910**

### CODE REVIEW

File: lib/hammer_cli_katello/activation_key.rb [hunk 1]
```diff
@@ -146,14 +146,19 @@ class SubscriptionsCommand < HammerCLIKatello::ListCommand
...
+        field :quantity_attached, _("Quantity")
         field :start_date, _("Start Date"), Fields::Date
         field :end_date, _("End Date"), Fields::Date
         field :support_level, _("Support")
         field :contract_number, _("Contract")
         field :account_number, _("Account")
       end
 
+      def extend_data(data)
+        data["format_consumed"] = data["quantity"] == -1 ? _("#{data['consumed']} out of Unlimited") : data["quantity"]
```
Comment (akofink) [inline lib/hammer_cli_katello/activation_key.rb]: This isn't quite right. `data["quantity"]` is the total, so this column should be more like this:

```rb
data["format_consumed"] = _("#{data['consumed']} out of #{data['quantity'] == -1 ? 'Unlimited' : data['quantity']}")
```

**Comment by vision2910** [comment]
@akofink thanks for the review. Your suggestion sounds good.

**Comment by akofink** [comment]
@vision2910 this PR is good to go except one rubocop failure. Any updates?

**Comment by vision2910** [comment]
> @vision2910 this PR is good to go except one rubocop failure. Any updates?

Fixed rubocop reported issue.

**Pull request merged**